document_page/models/document_page.py

```diff
@@ -64,7 +64,6 @@ class DocumentPage(models.Model):
         "document.page.history",
         "page_id",
         "History",
-        order="create_date DESC",
```

Comment:
Why do you remove this? Could you explain a bit?

Reply:
Hi @joao-p-marques ,
1. AFAIK the param `order` itself does not work in the field definition
2. The given model has defined default order by `id DESC` [here](https://github.com/dzungtran89/knowledge/blob/0874406449446cc2567e9e45ee01ab86a112cd40/document_page/models/document_page_history.py#L14), which does the same purpose

Reply:
Ok, thanks :+1: